tests_v2/test_bell_tab_indicator.py

```
@@ -0,0 +1,205 @@
+
+sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
+
+from cmux import cmux, cmuxError
+
+
+def surface_id_for_index(client: cmux, index: int) -> str:
+    surfaces = client.list_surfaces()
+    for entry in surfaces:
+        if entry[0] == index:
+            return entry[1]
+    raise RuntimeError(f"Surface index {index} not found")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)